Makefile

```diff
@@ -93,7 +93,7 @@ compile_notebooks:
 		if [ -f $$TARGETNAME ]; then \
 			echo $$TARGETNAME exists. Skipping compilation of $$BASENAME in Makefile. ; \
 		else \
-			python $(MD2IPYNB) $$BASENAME ; \
+			python $(MD2IPYNB) $$BASENAME True; \
```

Comment:
Here I want to pass one more argument to the python script. But it seems that it is not working currently. Could you please give some suggestions over this? @leezu

Reply:
sys.argv gives you strings. The string "False" will also have a boolean value of true.